vendor.conf

```diff
@@ -43,7 +43,7 @@ github.com/rancher/norman                     2da1bd2ffcacdef4bdf7e2c193d32f3b04
 github.com/rancher/kontainer-engine           95afcc99c7958e84122f011e610ff380d3555cf3
 github.com/rancher/rke                        f36020741696d687576b8d42bb1efa07f08c6184
 github.com/rancher/kontainer-driver-metadata  949885dc589573e2fd8ee3a72ec70b6c5226384d
-github.com/rancher/types                      8070d48db429f8f7957f666878509d7be4626628
+github.com/rancher/types                      4da53f80b53a679ba8b5b2e233f3e9f5fa245c86 https://github.com/nlowe/types
```

Comment:
Using `github.com/rancher/types` from my fork until rancher/types#902 is merged.